app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
Hold up a bit - I'm not even touching on the idea of replacing any of the 1001 uses of content loaded forms in the system - I'm just talking about the search/filtering forms, which _from what I could tell_ are actually implemented in a fairly separate way.

When it comes to this particular use case, neither of the existing methods _quite_ covered this use case, so whatever I did I would have to "break new ground" with this code. And if in two months it's decided to extend the brief search to be search-api based, sure, if it's decided the other way's better (or just lower immediate effort), this can be binned. At least this being here would be a strawman example of how such a form could operate.

And it's not really about _personal preference_, it's more that if I write code and commit it, I'm putting my personal seal-of-approval on it, and having seen so many subtle mistakes being made (and having made _so many_ mistakes myself) doing manual form processing, I'd find it a lot trickier to do that if I just mangled the existing methods.

("Personal seal-of-approval" sounds more pompous than I mean it - it's just an "I think this works ok" which I'd like people to be able to trust to a degree)

Unfortunately @tombye has just gone on hols...